src/pages/index.js

```diff
@@ -6,7 +6,12 @@ import { listings } from "@/data/mockListings";
 import FooterBar from "@/components/common/FooterBar";
 import NavBar from "../components/Header/NavBar";
 
+
```

Comment:
Please get rid of these extra lines. Also update this branch with `main` because the `<FooterBar>` component is already present in `index.js`.